### ISSUE TITLE
Can't define a not null ForeignKey

### ISSUE THREAD

**Issue by davidolrik** [piccolo-orm/piccolo#207]
Given the default schema with one little tweak, not allowing a band without a manager like this:

```python
from piccolo.table import Table
from piccolo.columns import ForeignKey, Integer, Varchar


class Manager(Table):
    name = Varchar(length=100)


class Band(Table):
    name = Varchar(length=100)
    manager = ForeignKey(references=Manager, null=False)
    popularity = Integer()
```

This breaks migrations as there is a default value of `None`:

```
ValueError: A default value of None isn't allowed if the column is not nullable.
```

Maybe this default should be removed from `ForeignKey()`.

**Comment by dantownsend** [comment]
Thanks for reporting this - I'll take a look.

**Comment by backwardspy** [comment]
i have just run into this same error when trying to add a mandatory foreign key field. i don't mind looking into the issue myself, any thoughts or pointers you can offer in the right direction would be super appreciated!

**Comment by dantownsend** [comment]
This should be fixed now in the latest version on PyPI (0.69.4).

@backwardspy Thanks for offering to fix it. It just happened that the fix was already underway in a separate issue, otherwise would've been much appreciated. 🙏
